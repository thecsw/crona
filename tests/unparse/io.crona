// Handles
// FromConsoleStmtNode
// ToConsoleStmtNode

fn : void () {
    write "Hello world!\nWhat is your name?\n";
    var1 : string;
    read var1;
    write "Hi ";
    write var1;
    write "!\n";
}