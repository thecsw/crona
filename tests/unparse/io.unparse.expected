fn:void(){
	write "Hello world!\nWhat is your name?\n";
	var1:byte array[0];
	read var1;
	write "Hi ";
	write var1;
	write "!\n";
}
